.travis.yml

```diff
@@ -26,7 +17,7 @@ env:
   - VERSION="10.0" TESTS="0" LINT_CHECK="0"
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="vauxoo/odoo"
```

Comment:
you do not need to change this to vauxoo/odoo only the main app repository is the one there.